crypto/crypto/src/hash.rs

```diff
@@ -113,10 +113,10 @@ use static_assertions::const_assert;
 use std::{self, convert::AsRef, fmt, str::FromStr};
 use tiny_keccak::{Hasher, Sha3};
 
-/// A prefix used to begin the salt of every libra hashable structure. The salt
+/// A prefix used to begin the salt of every diem hashable structure. The salt
 /// consists in this global prefix, concatenated with the specified
 /// serialization name of the struct.
-pub(crate) const LIBRA_HASH_PREFIX: &[u8] = b"LIBRA::";
+pub(crate) const DIEM_HASH_PREFIX: &[u8] = b"DIEM::";
```

Comment:
Are we still making hard backwards incompatible breaking changes? This is one of them...

Reply:
Yeah, trying to do it all at once.